site/using-weave.md

```diff
@@ -21,7 +21,7 @@ On `$HOST1` run:
 
     host1$ weave launch
     host1$ eval $(weave env)
-    host1$ docker run --name a1 -ti ubuntu:trusty
+    host1$ docker run --name a1 -ti weaveworks/ubuntu
```

Comment:
Since this is likely the first place where users will encounter that image, I suggest adding some explanation below (or linking to it).